Makefile

```diff
@@ -116,16 +116,33 @@ install-container-test:
 	docker build -t leapp-tests -f res/docker-tests/Dockerfile.$(subst :,,${CONTAINER}) res/docker-tests
 
 install-test:
+ifeq ($(shell id -u), 0)
```

Comment:
I tried to use this `` instead of shell, but that doesn't work

Reply:
You should really use backtick in makefiles instead of this.